server/game/gamesteps/triggeredabilitywindow.js

```diff
@@ -50,8 +58,16 @@ class TriggeredAbilityWindow extends BaseAbilityWindow {
             if(abilityChoice.text !== 'default') {
                 title += ' - ' + abilityChoice.text;
             }
+
             return { text: title, method: 'chooseAbility', arg: abilityChoice.id, card: abilityChoice.card };
         });
+
+        if(this.isCancellableEvent(player)) {
+            buttons.push({ timer: true, method: 'pass', id: uuid.v1() });
```

Comment:
I plan to refactor the whole way prompts and buttons work in the not too distant future so didn’t want to delve into that as part of that